packages/electron-builder/templates/nsis/boringInstaller.nsh

```diff
@@ -8,9 +8,9 @@
 !ifndef BUILD_UNINSTALLER
   Function StartApp
     ${if} ${Updated}
-      ${StdUtils.ExecShellAsUser} $0 "$startMenuLink" "open" "--updated"
+      ${StdUtils.ExecShellAsUser} $0 "$INSTDIR\${APP_EXECUTABLE_FILENAME}" "open" "--updated"
     ${else}
-      ${StdUtils.ExecShellAsUser} $0 "$startMenuLink" "open" ""
+      ${StdUtils.ExecShellAsUser} $0 "$INSTDIR\${APP_EXECUTABLE_FILENAME}" "open" ""
```

Comment:
Thanks.
I'll update the patchset (no option will be added. I'll just fallback to the executable in case the shortcut does not exist)